### PR TITLE
Create is_script.mdx

### PR DESCRIPTION
**Pull request by kbinreallife**
### Problem

scripts.lib page for is_script has not been created

### CODE REVIEW

File: docs/scripting/scripts.lib/is_script.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### Parameters
+
+# scriptname
+
+The string representation of a scriptname, e.g. "scripts.trust"
+
+### Return
+
+Returns a string
```
Comment (seanmakesgames) [inline docs/scripting/scripts.lib/is_script.mdx:21]:
```suggestion
Returns a boolean.
```

File: docs/scripting/scripts.lib/is_script.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Example
+
+```js
+function(context, args) {
+  const l = #fs.scripts.lib();
+  const scriptname = "scripts.trust";
+
+  return l.is_script(scriptname);
```
Comment (seanmakesgames) [inline docs/scripting/scripts.lib/is_script.mdx:30]:
```suggestion
  return l.is_script(context, arg) ? "this was called with is_script or was called as a subscript" : "etc";
```

**Review by seanmakesgames — COMMENTED** [review]
comments from office hours

**Review by matr1x-hackmud — APPROVED** [review]

File: docs/scripting/scripts.lib/is_script.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Returns a boolean.
+
+## Example
+
+```js
+function(context, args) {
+  const l = #fs.scripts.lib();
+
+  return l.is_script(context, arg) ? "this was called with is_script or was called as a subscript" : "etc";
```
Comment (seanmakesgames) [inline docs/scripting/scripts.lib/is_script.mdx:33]:
```suggestion
  return l.is_script(context, arg) ? "this was called with is_script:true or was called as a subscript" : "neither";
```

**Pull request merged**